modules/media.js

```diff
@@ -75,7 +78,14 @@ Media.prototype.clean = function(type, oncomplete) {
   }, function(error) {
     if (error) {
       errors.push(error);
+    } else {
+      ffosbr.history.set(type, {
+        title: capitalize(type),
```

Comment:
Is this a safe generalization if we have data type names with multiple words (e.g. "systemSettings")?